gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/ddl/catalog/GfxdSystemProcedures.java

```diff
@@ -1583,7 +1584,7 @@ public static void DROP_SNAPPY_TABLE(String tableIdentifier,
     LeadNodeSmartConnectorOpContext ctx = new LeadNodeSmartConnectorOpContext(
         LeadNodeSmartConnectorOpContext.OpType.DROP_TABLE,
         tableIdentifier, null, null, null, null, null, true, ifExists,
-        null, null, null, null, null, null);
+        null, null, null, null, null, null,false, null, null, false);
```

Comment:
We should refactor this to better handle the context with less and manageable numbe rof arguments. @dshirish has a todo here.